sis_provisioner/utils.py

```diff
@@ -51,8 +51,8 @@ def get_class_desc(class_code, majors):
     if class_code not in getattr(settings, 'INCLUDE_CLASS_CODES', []):
         return None
 
-    if (any('MBA' in major.major_abbr_code for major in majors) and
-            get_synced_college_name(majors) == 'Foster School of Business'):
+    if (any('MBA' in major.major_abbr_code and major.college == 'E'
```

Comment:
The example csv seems to show that a student should show an MBA instead of Masters if they have an MBA-like major, even if it doesn't take priority